website/docs/r/catalog.html.markdown

```diff
@@ -47,6 +47,7 @@ The following arguments are supported:
 * `refresh` - (Optional) Catalog will wait for refresh after tf creation and on every tf read. Default `false` (bool)
 * `scope` - (Optional) The scope of the catalog. `cluster`, `global`, and `project` are supported. Default `global` (string)
 * `username` - (Optional/Sensitive) The username to access the catalog if needed (string)
+* `version` - (Optional/ForceNew) Helm version for the catalog. Available options: `helm_v2` (default) and `helm_v3` (string)
```

Comment:
Are these the options from the Rancher API?

Reply:
Yes, these options comes from Rancher API